search/src/Components/upload/uploader_single.js

```diff
@@ -9,6 +9,7 @@ const UploaderSingle  = (props) => {
     const getUploadParams = ({ file, meta }) => {
         const body = new FormData()
         body.append('weedcoco', file)
+        body.append('schema', props.schema)
```

Comment:
This is okay for now, but I think that upload format is a more relevant parameter going forward.